### ISSUE TITLE
Allow user to specify date range instead of a single month

### ISSUE THREAD

**Issue by lakikowolfe** [FredHutch/coopMetrics#1]
**Current**
The package is set up to pull a single month at a time. To pull multiple months requires user to iterate over the function.

**Ideal**
User specifies a date range and interval (weeks, months, years) and is returned a dataframe with the metrics as variables and each month/week/year as rows

**Comment by lakikowolfe** [comment]
implement date ranges for commits using the `since` and `until` params

https://docs.github.com/en/rest/reference/repos#list-commits

**Comment by lakikowolfe** [comment]
**This is what the code will look like to pull a specific date range of commits**

```
tst1 <- gh("GET /repos/:owner/:repo/commits",
           owner = "FredHutch",
           repo = "coop",
           since = begin,
           until = end)
```

Current code uses the `commit_activity` query which pulls the last year of commit activity in a repo grouped by week.

**Need to rethink how/when to pull/subset the dataset more efficiently**

**Comment by lakikowolfe** [comment]
This is the code to pull a specific date range / interval for google analytics

```
metrics <- c("users", "newUsers", "sessions")
dimensions <- c("month", "week", "year")
google_analytics(viewId,
                 date_range = dateRange,
                 metrics = metrics,
                 dimensions = dimensions,
                 anti_sample = TRUE)

```

**Comment by lakikowolfe** [comment]
resolved in #10
